### PR TITLE
#14/Game mode name

### LINKED ISSUE #14: UI Komponent | Nazwa trybu gry
### Kryteria akceptacji

Należy zweryfikować i skonsultować, a także ewentualnie uaktualnić przed rozpoczęciem prac nad zgłoszeniem.

- [x]  Komponent zdefiniowany w odrębnym pliku .js

- [x]  Komponent przyjmuje na wejściu opisową nazwę trybu gry, np. 'Who is this character?'

- [x]  Komponent wyświetla przekazany napis po napisie MODE.

- [x]  Działanie komponentu zostało pokryte testami jednostkowymi.

- [x]  Wygląd komponentu wg. projektu:

### PR DESCRIPTION
**Pull request by KlaudiaDabrowska**
closes #14 

Creating a simple function that selects a question based on the game mode. 
Adding a game mode question to the html element.

### CODE REVIEW

File: src/app/questionForMode.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const questionForMode = (mode) => {
+  const questions = {
```
Comment (vieraboschkova) [inline src/app/questionForMode.js:2]: Perhaps you could add some validation? What if the mode comes in as undefined or any other value? What should happen then?

File: src/app/questionForMode.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    People: 'Who is this character?',
+    Houses: 'What is this house?',
+    Wands: 'Who this wand belongs to?',
+  };
+
+  const modeQuestion = `MODE: ${questions[mode]}`;
+
+  document.querySelector('.game__mode').textContent = modeQuestion;
```
Comment (vieraboschkova) [inline src/app/questionForMode.js:10]: Would be great if you would provide a way to test your code.

Comment (vieraboschkova) [inline src/app/questionForMode.js:10]: The part of the task is also styling the mode question, perhaps you could give it a thought? It is a container with text abd this text might be the only thing about it that is different from other (maybe its a specfic heading), so perhaps you should also think of it not as a single mode question, but one of a group of containers, so you could reuse the same styling for mode rules or maybe even buttons? Something to think about.

**Review by Smyku6 — APPROVED** [review]
Nice job!

**Pull request merged**